Certification-Standard/introduction.md

```diff
@@ -0,0 +1,239 @@
+# Cardano Certification Standard
+
+**Draft version**
+**Last modification September 21th 2023**
+**Authors:**
+
+- Romain Soulat
+
+## 1 Introduction
+
+The Cardano Certification Standard (CCS) is a framework designed to facilitate the comparability of independent evaluations within the Cardano blockchain ecosystem. This standard establishes a comprehensive set of requirements for assessing the security features of Decentralized Applications (DApps) on the Cardano blockchain, as well as the assurance measures applied during security evaluations.
+
+The primary objective of the evaluation process is to establish a level of confidence that the security functionalities of these DApps and the assurance measures align with the predefined requirements. Consequently, the evaluation results serve as a valuable resource for end-users seeking to gauge whether a particular DApp aligns with their security requirements and expectations.
+
+The Cardano Certification Standard also serves as a guide for both DApp developers and evaluators. It offers insights and best practices for the development of DApps with robust security features and provides a structured approach for evaluating their security posture.
+
+Flexibility is a core principle of the Cardano Certification Standard. It is designed to accommodate a diverse range of evaluation methodologies, ensuring that all essential security properties required by a DApp can be effectively assessed. However, it is essential to note that the validity of an evaluation is contingent upon the specific security properties evaluated and the methods employed. Auditors and users are advised to examine the DApp's security requirements and evaluation methodologies to determine the meaningfulness of the evaluation results.
+
+The Cardano Certification Standard aims at adressing:
+
+- The protection of assets of any user interacting with the DApp
+- The loss of use of the DApp
```

Comment:
I was trying to reformulate this. It's something like "Ensuring the accessibility and liveness of the  services that the DApp provides."